workflow/rocoto/workflow_tasks.py

```diff
@@ -510,8 +510,23 @@ def aeroanlfinal(self):
 
         return task
 
+    def preplandobs(self):
+
+        deps = []
+        dep_dict = {'type': 'task', 'name': f'{self.cdump}prep'}
+        deps.append(rocoto.add_dependency(dep_dict))
+        dependencies = rocoto.create_dependency(dep=deps)
+
+        resources = self.get_resource('preplandobs')
+        task = create_wf_task('preplandobs', resources, cdump=self.cdump, envar=self.envars, dependency=dependencies,
+                              cycledef=f'{self.cdump}_land_prep')
+
+        return task
+
     def landanlinit(self):
 
+        # TODO: Q: what from gdaspost -6H or atmf009.nc does landanlinit depend on?
```

Comment:
@aerorahul I just followed the settings from `aeroanlinit`. I think the `landanlinit` may just require model background from `prep`. @CoryMartin-NOAA Do you have any suggestions? Thanks.

Reply:
@jiaruidong2017 I believe it depends on the 6 hour RESTART files, no? which I guess would be the atmf009 for simplicity sake

Reply:
Ok. Thanks.
Completion of `gdasprep` should be enough to capture that, instead of the data dependency on `atmf009.nc` from previous cycle.
I will update the dependencies.

I assume this will also be true for the `aeroanlinit` @CoryMartin-NOAA?

Reply:
@CoryMartin-NOAA Yes, it depends on the 6-hr RESTART files.

Reply:
I have resolved and simplified the dependencies for both `landanlinit` and `aeroanlinit`.